app/views/layouts/application.html.erb

```diff
@@ -17,7 +17,7 @@
   <%= csrf_meta_tags %>
 </head>
 <body>
-  <%# https://github.com/seyhunak/twitter-bootstrap-rails#using-helpers %>
+<%# https://github.com/seyhunak/twitter-bootstrap-rails#using-helpers %>
```

Comment:
Just to mention ist: in my user story, i gave a fluid width to the navbar. I will commit it later on.